### PR TITLE
fix: Configure the PortalNavigationIconUpgradePlugin to set icons to the portal navigation nodes - EXO-69848

### PR DESCRIPTION
**Pull request by sofyenne**
Prior to this change after migrating from the 6.4 to the 6.5, the external stream node navigation icon was incorrect , this change is going to configure the PortalNavigationIconUpgradePlugin in order to set icons for the portal navigation nodes.

### CODE REVIEW

File: data-upgrade-navigations/src/main/resources/conf/portal/configuration.xml [hunk 1]
```diff
@@ -195,5 +195,48 @@
...
+      <init-params>
+        <value-param>
+          <name>product.group.id</name>
+          <description>The groupId of the product</description>
+          <value>org.exoplatform.platform</value>
+        </value-param>
+        <value-param>
+          <name>plugin.upgrade.target.version</name>
+          <description>The plugin target version (will not be executed if previous version is equal or higher than 6.6.0)</description>
+          <value>6.5.2</value>
```
Comment (azayati) [inline data-upgrade-navigations/src/main/resources/conf/portal/configuration.xml:212]: why it is not 6.6.0 ?

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by azayati** [comment]
@sofyenne have you tested that on pgsql ?

**Review by azayati — CHANGES_REQUESTED** [review]
@sofyenne have you tested that on pgsql ?

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Comment by sofyenne** [comment]
> @sofyenne have you tested that on pgsql ?

Yep!

File: data-upgrade-navigations/src/test/java/org/exoplatform/migration/PortalNavigationIconMigrationTest.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
+package org.exoplatform.migration;
```
Comment (azayati) [inline data-upgrade-navigations/src/test/java/org/exoplatform/migration/PortalNavigationIconMigrationTest.java:17]: put the header

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by Jihed525 — DISMISSED** [review]

**Review by Jihed525 — APPROVED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**